01-intro.Rmd

```diff
@@ -37,24 +59,82 @@ The major requirement of the policy is that all grant proposals (submitted after
 
 For certain grant mechanisms for projects that do not generate data, compliance with the policy is not required. For certain types of data, sharing is not possible, and a justification will be required instead. 
 
-What about grant renewals? 
+The following outline several key questions:
+
+- **Does my research generate scientific data?** If yes, you will need to submit a DMS Plan.
+- **When do I need to share my data?**
+- **Can I elect to not share data?** The short answer is yes. However, you must still justify this choice in a submitted DMS Plan.
+
+## Does my research generate scientific data?
+
+The NIH Data Management and Sharing (DMS) Policy applies to **all NIH-supported research generating scientific data**. But what is "scientific data"?
+
+**SCIENTIFIC DATA**
+
+Scientific data are the “recorded factual material of sufficient quality to validate and replicate research findings, regardless of whether the data are used to support scholarly publications”. This can include any of the following if they are applicable to your study:
+
+- unpublished results
+- null results
+- results used to publish papers
+<!-- TODO: are the following two points correct? -->
```

Comment:
I think this is a question we're going to get, but I don't know the answer for raw or cleaned data.

Reply:
right... it's an important question

Reply:
watching it again... I think it is whatever you are able to share - so if you can share raw I think they prefer that - but maybe not required??? I think we might need to ask